docs/HOSTNAMED.md

```diff
@@ -0,0 +1,121 @@
+---
+title: On hostnamed
+category: Documentation for Developers
+layout: default
+SPDX-License-Identifier: LGPL-2.1-or-later
+---
+
+**This page has been obsoleted and replaced:** [ https://www.freedesktop.org/software/systemd/man/org.freedesktop.hostname1.html](https://www.freedesktop.org/software/systemd/man/org.freedesktop.hostname1.html).
```

Comment:
We don't want to "port" any pages which have already been obsoleted. So please just drop any which have this header from the pull request.